autogen/bundlegroups/sc.fiji.xml

```diff
@@ -0,0 +1,21 @@
+<!-- trove4j bundle -->
+<bundlegroup name="io.scif">
+
+	<!-- SCIFIO -->
+	<bundle name="fiji-bigdataviewer" version="${bigdataviewer-core.version}">
```

Comment:
do we want the bundle to be named fiji-bigdataviewer? because actually the bdv has nothing to do with fiji directly.
Furthermore is the <!--- SCIFIO ---> comment wrong